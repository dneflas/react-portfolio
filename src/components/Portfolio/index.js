import React from "react";
import Project from "../Project";

function Portfolio() {
  return (
    <section className="my-5" id="portfolio">
      <h2 data-testid="portfolioH2">Portfolio</h2>
      <Project></Project>
    </section>
  );
}

export default Portfolio;
