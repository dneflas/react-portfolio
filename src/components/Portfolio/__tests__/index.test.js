import React from "react";
import { render, cleanup } from "@testing-library/react";
import "@testing-library/jest-dom/extend-expect";
import Portfolio from "..";

afterEach(cleanup);

describe("portfolio section renders", () => {
  it("renders", () => {
    render(<Portfolio />);
  });
  it("matches snapshot", () => {
    const { asFragment } = render(<Portfolio />);
    expect(asFragment()).toMatchSnapshot();
  });
});

describe("title renders", () => {
  it("h2 renders", () => {
    const { getByTestId } = render(<Portfolio />);
    expect(getByTestId("portfolioH2")).toHaveTextContent("Portfolio");
  });
});
