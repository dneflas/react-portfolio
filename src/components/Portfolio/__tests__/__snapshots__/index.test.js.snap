// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`portfolio section renders matches snapshot 1`] = `
<DocumentFragment>
  <section
    class="my-5"
    id="portfolio"
  >
    <h2
      data-testid="portfolioH2"
    >
      Portfolio
    </h2>
    <div
      class="flex-row justify-space-between"
    >
      <div
        class="col-12 col-md-6 mb-2 bg-secondary text-dark p-3 flex-column border-round projects"
        style="background-image: url(0.png);"
      />
      <div
        class="col-12 col-md-6 mb-2 bg-secondary text-dark p-3 flex-column border-round projects"
        style="background-image: url(1.png);"
      />
      <div
        class="col-12 col-md-6 mb-2 bg-secondary text-dark p-3 flex-column border-round projects"
        style="background-image: url(2.png);"
      />
      <div
        class="col-12 col-md-6 mb-2 bg-secondary text-dark p-3 flex-column border-round projects"
        style="background-image: url(3.png);"
      />
      <div
        class="col-12 col-md-6 mb-2 bg-secondary text-dark p-3 flex-column border-round projects"
        style="background-image: url(4.png);"
      />
      <div
        class="col-12 col-md-6 mb-2 bg-secondary text-dark p-3 flex-column border-round projects"
        style="background-image: url(5.png);"
      />
    </div>
  </section>
</DocumentFragment>
`;
